packages/core/src/components/meter/meter.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, Element, h, Host, Prop } from '@stencil/core'
+
+@Component({
+  tag: 'atom-meter',
+  styleUrl: 'meter.scss',
+  shadow: true,
+})
+export class AtomMeter {
+  @Prop() type?: 'success' | 'neutral' | 'warning' | 'danger' = 'neutral'
+  @Prop() size?: 'small' | 'large' = 'large'
+  @Prop() title: string
+  @Prop() centerTitle?: boolean = false
+  @Prop() min: number = 0
+  @Prop() max: number
+  @Prop() actual: number
+
+  @Element() el: HTMLElement
+
+  private isMobile = () => {
+    return window.matchMedia('(max-width: 768px)').matches
```

Comment:
there is a utils folder in the project, shall you include this there, man?

Reply:
Excellent idea! I created the function and also replaced it in the tooltip, it was also used haha

Made: https://github.com/juntossomosmais/atomium/pull/695/commits/9ac14c4d857407de46c65757a31a605ee2b95255